### ISSUE TITLE
Subcategory as separate key

### ISSUE THREAD

**Issue by groenroos** [amio/emoji.json#11]
It's handy having the subcategory included in the 'full' JSON, but putting it in the category string means it's much more difficult to separate the emojis into separated categories on user-facing applications (who want the major categories, but likely not so much the subcategories). Could the subcategory be put into a separate `subcategory` property?

**Comment by groenroos** [comment]
I agree that `group` and `subgroup` would be better for consistency, and that that should be introduced in 13.x.x since it would be an incompatible change.

Filesize is becoming increasingly irrelevant these days, but there's always [JSONH](https://github.com/WebReflection/JSONH) for those worried about it! 🙂

**Comment by amio** [comment]
Let's add `group` and `subgroup` now and,
- [ ] remove `category` in v13
